drivers/power_domain/power_domain_psl.c

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2022 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nuvoton_npcx_power_domain_psl
+
+#include <kernel.h>
+#include <drivers/gpio.h>
+#include <drivers/pinctrl.h>
+#include <pm/device.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(power_domain_psl, LOG_LEVEL_INF);
```

Comment:
Default log to CONFIG_POWER_DOMAIN_LOG_LEVEL?

Reply:
Done.